examples/prisma.md

```diff
@@ -1,68 +1,57 @@
 # Using Prisma on Cloudflare Workers
 
-There are two ways to use Prisma with Cloudflare Workers: using Prisma Accelerate or using the Driver Adapter.
+[Prisma ORM](https://www.prisma.io/docs) provides a modern, robust toolkit for interacting with databases, and when paired with Hono and Cloudflare Workers, it enables you to deploy high-performance, serverless applications at the edge.
 
-## Using Prisma Accelerate
+In this guide, we’ll walk you through integrating Prisma using both [Prisma Postgres](https://www.prisma.io/postgres)—a managed, serverless PostgreSQL database—and [Driver adapters](https://www.prisma.io/docs/orm/overview/databases/database-drivers#driver-adapters), giving you the flexibility to choose the best approach for your project.
```

Comment:
this almost reads to me as if we're doing both: using PPg _and_ driver adapters. can you make it more clear that these are two distinct approaches? and also highlights the differences between the two (i.e. that driver adapters is an approach that enables using any DB supported by Prisma ORM)